src/posix/platform/README_RCP_CAPS_DIAG.md

```diff
@@ -6,10 +6,22 @@ This module provides diag commands for checking RCP capabilities.
 
 ## Command List
 
+- [perf](#perf)
 - [spinel](#spinel)
 
 ## Command Details
 
+### perf
+
+Check the RCP performance.
```

Comment:
Is this really testing functionality or performance?

Reply:
I have updated the command name to `srcmatchtable`.